lib/tasks/update_received_quantity_to_package.rake

```diff
@@ -0,0 +1,15 @@
+# rake goodcity:update_received_quantity_to_package
+namespace :goodcity do
+  desc 'Update received quantities to package'
+  task update_received_quantity_to_package: :environment do
+    counter = 0
+    quantity_packages = Package.where("packages.received_quantity > 1")
```

Comment:
Package.where("packages.received_quantity > 1").find_each will be better here. No need assign it to variable. :)

Reply:
sure 👍

Reply:
yes, and definitely use `find_each` - worth a quick read to see why. hint: batch loading saves memory -  just using `find` will load 300,000+ records from SQL into ruby serialized AR objects before it's started to process the loop. See https://apidock.com/rails/ActiveRecord/Batches/ClassMethods/find_each